runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -1218,9 +1217,14 @@ NNFW_STATUS nnfw_session::output_tensorindex(const char *tensorname, uint32_t *i
 NNFW_STATUS nnfw_session::set_backends_per_operation(const char *backend_settings)
 {
   if (backend_settings == NULL)
-  {
     return NNFW_STATUS_ERROR;
-  }
-  _compiler->set_backend_from_str(backend_settings);
+
+  if (!isStateModelLoaded())
```

Comment:
`set_backends_per_operation` is allowed to set as long as it is in `LOADED` state.